test/es-module/test-esm-import-meta.mjs

```diff
@@ -3,7 +3,8 @@ import assert from 'assert';
 
 assert.strictEqual(Object.getPrototypeOf(import.meta), null);
 
-const keys = ['resolve', 'url'];
+
```

Comment:
⚙️ (optional) It seems like lack of linter config. 📓  https://eslint.org/docs/latest/rules/no-multiple-empty-lines
```suggestion
```

Reply:
ah, config is allowed to use 2 lines: https://github.com/nodejs/node/blob/86afefafdae2bb00e2f2e886d17ca077bde4cf53/.eslintrc.js#L195. sorry my bad.